### PR TITLE
Add descscription to client

### PR DESCRIPTION
**Pull request by LindaLawton**
**What issue does this PR address?**
[2280](https://github.com/IdentityServer/IdentityServer4/issues/2280)

**Does this PR introduce a breaking change?**

nope

**Please check if the PR fulfills these requirements**
- [x] The commit follows our [guidelines](https://github.com/IdentityServer/IdentityServer4/blob/dev/.github/CONTRIBUTING.md)
- [x] Unit Tests for the changes have been added (for bug fixes / features)

**Other information**:

Made some changes to the consent screen to display this new info in the Host project.  The rest should be pretty basic.

### CODE REVIEW

File: src/Host/Views/Consent/Index.cshtml [hunk 1]
```diff
@@ -8,9 +8,19 @@
                 <div class="client-logo"><img src="@Model.ClientLogoUrl"></div>
```
Comment (LindaLawton) [inline src/Host/Views/Consent/Index.cshtml]: Note.   I have urls inserted here and this isnt working due to CORS errors.    Not sure what you have been inserting.

File: src/Host/Views/Consent/Index.cshtml [hunk 2]
```diff
@@ -7,10 +7,12 @@
             {
                 <div class="client-logo"><img src="@Model.ClientLogoUrl"></div>
             }
-            <h1>
-                @Model.ClientName
-                <small>is requesting your permission</small>
-            </h1>
+            <h1>@Html.Raw((string.IsNullOrWhiteSpace(Model.ClientUrl))? $"{@Model.ClientName}" : $"<a href='{@Model.ClientUrl}'>{@Model.ClientName}</a>")                </h1>
```
Comment (brockallen) [inline src/Host/Views/Consent/Index.cshtml]: I don't like Html.Raw in here -- XSS potential.

Comment (LindaLawton) [reply src/Host/Views/Consent/Index.cshtml]: I haven't done web development in 10 years so i am a bit behind in what one should be using.

Do you have a suggestion?

Comment (brockallen) [reply src/Host/Views/Consent/Index.cshtml]: I'd just write the if/else blocks to build the html.

Comment (LindaLawton) [reply src/Host/Views/Consent/Index.cshtml]: I wont have time to do this until Thursday.  Up to you if you can wait it looks like your on a clean out PR spree today.

Comment (brockallen) [reply src/Host/Views/Consent/Index.cshtml]: No, I am just looking at issues. No time yet for me to do any code :)

**Comment by brockallen** [comment]
Given that we reworked the models last week, I'll close this and add the description manually. Thanks.

**Pull request closed without merging**